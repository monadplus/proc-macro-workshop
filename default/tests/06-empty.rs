#[derive(Debug, PartialEq, Eq, derive_default::Default)]
enum Empty{}

fn main() {}
