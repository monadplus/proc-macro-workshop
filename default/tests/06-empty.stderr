error: #[derive(Default)] is not supported for empty enums
 --> tests/06-empty.rs:2:6
  |
2 | enum Empty{}
  |      ^^^^^
